app/webpack/stylesheets/components/_notification_banner.scss

```diff
@@ -0,0 +1,21 @@
+.govuk-notification-banner--failure {
+  border-color: $govuk-error-colour;
+  background-color: $govuk-error-colour;
+
+  .govuk-notification-banner__link:link,
+  .govuk-notification-banner__link:visited {
+    color: $govuk-error-colour;
+  }
+
+  .govuk-notification-banner__link:hover {
+    color: govuk-shade($govuk-error-colour, 20%);
+  }
+
+  .govuk-notification-banner__link:active {
+    color: $govuk-error-colour;
+  }
+
+  .govuk-notification-banner__link:focus {
+    color: $govuk-text-colour;
+  }
```

Comment:
I would recommend using the [govuk-link-style-error mixin](https://github.com/alphagov/govuk-frontend/blob/main/package/govuk/helpers/_links.scss#L130) to keep the style consistent.